addons/edi/models/edi_partner_record.py

```diff
@@ -47,6 +47,7 @@ class EdiPartnerRecord(models.Model):
     partner_id = fields.Many2one('res.partner', string="Partner",
                                  required=False, readonly=True, index=True,
                                  auto_join=True)
+    parent_id = fields.Many2one('res.partner', string="Parent", readonly=True, index=True)
```

Comment:
Why is this needed?

Reply:
There are tests in `edi_sale` which reference this `parent_id` field that previously didn't exist in this repo.

This field was added in by another custom module at some point, but I don't know why it wasn't added in one of the EDI modules originally.

`parent_id` seemed generic enough to me for it to be included in `edi` and not just `edi_sale`.

Reply:
Sorry should be more clear, the tests fail because `remove_sales_for_invalid_partner_updates` in `edi_sale` references this `parent_id` field.

https://github.com/unipartdigital/odoo-edi/blob/master/addons/edi_sale/models/edi_sale_request_document.py#L127

Reply:
Wouldn't the correct thing to do to the only override that method in the custom module otherwise we just end up bloating edi_partner_record. I know its only one field but a parent of a partner is not really part of most partners.

Reply:
Refactor this out into an `get_invalid_partners` then its easier
https://github.com/unipartdigital/odoo-edi/blob/master/addons/edi_sale/models/edi_sale_request_document.py#L126

Reply:
Yeah good idea, will go look into that. :)